cranelift-codegen/meta/src/cdsl/xform.rs

```diff
@@ -148,16 +153,17 @@ impl Transform {
 /// pool `var_pool`. If the variable was not present in the symbol table, then add it to the list of
 /// `defined_vars`.
 fn var_index(
-    name: &'static str,
+    name: impl Into<String>,
```

Comment:
Could this just be a `&str` instead? (in the spirit of reducing the number of type-parameterized functions in general)